programs/fileio.c

```diff
@@ -1585,7 +1585,7 @@ FIO_determineCompressedName(const char* srcFileName, const char* outDirName, con
 
 static size_t FIO_getLargestFileSize(const char** inFileNames, unsigned nbFiles)
 {
-    size_t i; size_t fileSize; size_t maxFileSize = 0;
+    size_t i, fileSize, maxFileSize = 0;
```

Comment:
Hmm. freeBSD tests failed after this. I'm just going to change this back to what I had before @Cyan4973 '

Reply:
This [multi-variables declaration](https://www.tutorialspoint.com/cprogramming/c_variables.htm) is standard C, 
so all compilers are expected to support it.

The freeBSD tests [work fine](https://github.com/facebook/zstd/runs/380215789).

The failing test was a flacky one, named "Minimal Decompressor Macros", on travisCI.
It's the first time I can remember seeing this specific test flacky, 
but unfortunately all tests can be occasionally flacky, 
so it's good to check the actual reason of the failure.

In this case, compilation failed while parsing a strange symbol in an unrelated file (unmodified in the commit).
Restarting the test, it now passes without issue : 
https://travis-ci.org/facebook/zstd/jobs/634432963?utm_medium=notification&utm_source=github_status

Reply:
Ah okay. Noted!